lib/Language/ASL/Formulas/Serialize.hs

```diff
@@ -460,7 +468,14 @@ expandSymFn sym symFn = case WB.symFnInfo symFn of
 -- given signature.
 uninterpFunctionMaker :: forall sym. WI.IsSymExprBuilder sym => sym -> FunctionMaker sym
 uninterpFunctionMaker sym = FunctionMaker sym $ \_nm sig -> do
-  Right <$> (liftIO $ WI.freshTotalUninterpFn sym (U.makeSymbol (T.unpack (fsName sig))) (fsArgs sig) (fsRet sig))
+  Right <$> (liftIO $ WI.freshTotalUninterpFn sym (U.makeSymbol (toUFIfNeeded (fsName sig))) (fsArgs sig) (fsRet sig))
+
+-- | If a function name doesn't already have a df_ or uf_ prefix, add uf_
+toUFIfNeeded :: T.Text -> String
+toUFIfNeeded t
+  | "uf_" `T.isPrefixOf` t = T.unpack t
+  | "df_" `T.isPrefixOf` t = T.unpack t
+  | otherwise = "uf_" ++ T.unpack t
```

Comment:
This is a bit suspicious. Since the spec isn't going through a formula printer/parser iteration any more we should have complete control over all the function names, and we should be able to ensure this case doesn't get hit (given the other changes, this should be mostly the case already).

Reply:
I think there are some prefixes getting added in other places inconsistently.  I didn't have a chance to really track them down.  Without that, I was getting multiple prefixes on a number of functions.